news/2018-04-22-beatmap-spotlights-march-2018.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "Beatmap Spotlights: March 2018"
+date: 2018-04-22 12:00:00 +0000
+---
+
+The March Spotlights are now available, bringing a selection of incredible maps for every game mode, and a shiny new medal for the collectors among us!
+
+![](/wiki/shared/news/2018-04-22-beatmap-spotlights-march-2018/spotlight-header.jpg)
+
+The beatmap charts for the March Spotlights can be found [here](https://osu.ppy.sh/p/chart?ch=MONTH1804) Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist?t=r) for easier access. **The March 2018 Spotlights rankings will end on 15th May 2018.**
+
+## Past Spotlights results
+
+First of all, congratulations to the winners of the February 2018 Spotlights. They receive one month of supporter status for their efforts!
++ **osu!:** [Toy](https://osu.ppy.sh/users/2757689)
++ **osu!taiko:** [Alepat](https://osu.ppy.sh/users/1164931)
++ **osu!catch:** [Guillotine](https://osu.ppy.sh/users/4365562)
++ **osu!mania:** [Cryolien](https://osu.ppy.sh/users/1626983)
+
+# Announcing changes in the prizes
+
+**We are happy to announce that from this month onward, we will be rewarding the Top 3 players of each Spotlights leaderboard with special user titles for their efforts!** 
+
+The titles are as follows:
+
++ **1st place:** Spotlights Grandmaster
++ **2nd and 3rd place:** Spotlights Master
```

Comment:
2nd and 3rd sharing the same titles seems.. off to me

the titles in general don't seem all that inspiring tbh, should at least be mode specific i think

Reply:
if need be we can go live w/o the titles and announce titles next month

Reply:
I feel like awarding the top 3, as you suggest, would be the best but finding awesome titles which has some kind of order is another matter. I wondered about copying shogi titles which are Kyoshi, Hanshi and Meijin but while the last one sounds good enough the rest is kinda ????

Or another possible ones are Cognoscent, Erudite, Illuminatus
Or Scholar, Adept, Master

I was thinking about these but I dunno, they kinda feel weird to me. 🤔

Reply:
it sounds weird because spotlights is plural, sounds better without the S